src/libcore/simd.rs

```diff
@@ -37,11 +24,33 @@
 #![unstable(feature = "core_simd",
             reason = "needs an RFC to flesh out the design",
             issue = "27731")]
+#![deprecated(since = "1.3.0",
+              reason = "use the external `simd` crate instead")]
 
 #![allow(non_camel_case_types)]
 #![allow(missing_docs)]
+#![allow(deprecated)]
+
+use ops::{Add, Sub, Mul, Div, Shl, Shr, BitAnd, BitOr, BitXor};
+
+macro_rules! argh {
```

Comment:
ABIs are verified as valid as soon as they are parsed, i.e. before `cfg` stripping. The `platform-intrinsic` ABI is new, so stage0 doesn't know about it, but it still errors out when it hits it (despite this being in a `cfg(not(stage0))` module). It's unfortunate, to say the least, and hence the name.

(